site/themes/getenvoy/layouts/partials/quickstart.html

```diff
@@ -3,7 +3,7 @@
     <div id="quickstart-cli">
       <h2>Fetch, run and debug Envoy using the CLI</h2>
       <pre>
-        <code class="language-sh">$ curl -L https://getenvoy.io/install.sh | bash -s -- -b /usr/local/bin <br />$ getenvoy run --version</code>
+        <code class="language-sh">$ curl -L https://getenvoy.io/install.sh | bash -s -- -b /usr/local/bin <br />$ getenvoy run --config-yaml "admin: {address: {socket_address: {address: '127.0.0.1', port_value: 0}}}"</code>
```

Comment:
I think we could use `9901 ` for clarity.. not need to assign random port on user side

Reply:
great idea